CHANGES.txt

```diff
@@ -1,6 +1,9 @@
 1.3.x (unreleased)
  * Removed use of Thrift describe_ring and replaced it with native Java Driver
    support for fetching TokenRanges (SPARKC-93)
+ * Decoupled PredicatePushDown logic from Spark (SPARKC-166)
```

Comment:
Yes, sounds charming, but isn't the quality/readability of the CHANGES.txt going to decrease? In CHANGES.txt we're often putting information what has *actually* been done, while JIRA often contains *what needs to be done*. Sometimes those things are different. For similar reason generating CHANGES.txt directly from the git-log is a bad idea.